cwl/src/test/scala/cwl/CommandLineToolSpec.scala

```diff
@@ -0,0 +1,23 @@
+package cwl
+
+import cwl.CommandLineTool.CommandInputParameter
+import org.scalatest.{FlatSpec, Matchers}
+
+class CommandLineToolSpec extends FlatSpec with Matchers {
+
+  behavior of "CommandLineTool"
+
+  it should "order input arguments when bound to command line" in {
+
+    val a = CommandInputParameter("a", inputBinding = None)
+    val b = CommandInputParameter("b", inputBinding = Some(CommandLineBinding()))
+
+    val ordered: Seq[CommandInputParameter] = CommandLineTool.orderedForCommandLine(Array(a,b))
+
+    ordered.contains(a) shouldBe false
+
+    ordered.contains(b) shouldBe true
```

Comment:
was this actually a test of ordering?

Reply:
No of filtering.  I'll fix it up

Reply:
Were you going to add an ordering test too?